src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseParser.cs

```diff
@@ -86,31 +86,10 @@ internal static class WinHttpResponseParser
             // Parse raw response headers and place them into response message.
             ParseResponseHeaders(requestHandle, response, stripEncodingHeaders);
 
-            // Store response header cookies into custom CookieContainer.
-            if (cookieUsePolicy == CookieUsePolicy.UseSpecifiedCookieContainer)
-            {
-                Debug.Assert(cookieContainer != null);
-
-                if (response.Headers.Contains(HeaderNameSetCookie))
-                {
-                    IEnumerable<string> cookieHeaders = response.Headers.GetValues(HeaderNameSetCookie);
-                    foreach (var cookieHeader in cookieHeaders)
-                    {
-                        try
-                        {
-                            cookieContainer.SetCookies(request.RequestUri, cookieHeader);
-                        }
-                        catch (CookieException)
-                        {
-                            // We ignore malformed cookies in the response.
-                        }
-                    }
-                }
-            }
-
             return response;
         }
 
+        // Returns the first header or throws if header isn't found.
```

Comment:
I remember seeing an email. There's an issue regarding public documentation here: https://github.com/dotnet/corefx/issues/230
It would be nice to understand what to use for internal function descriptions.